tools/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_swift_tool_subdirectory(swift-demangle)
 add_swift_tool_subdirectory(lldb-moduleimport-test)
 add_swift_tool_subdirectory(sil-extract)
 add_swift_tool_subdirectory(swift-llvm-opt)
+add_swift_tool_subdirectory(swift-api-digester)
 
 if(SWIFT_HOST_VARIANT STREQUAL "macosx")
   # Only build Darwin-specific tools when deploying to OS X.
```

Comment:
If we are trying to build swift-api-digester for all platforms, please remove this section.